### DISCUSSION TITLE
Nice Names in rules protected property

### DISCUSSION THREAD

**Discussion by ousid** [livewire/livewire#1903] [4. General]
### Description
The `$rules` protected Property is a powerful feature, but I Notice there is no  nice-names  option provides into it

### Exact steps to reproduce
- Create a new component
- Make Validation in `updated` method or `create` method for example
- Make nice-names instead of showing the default input names if there's an error

#### With `$rules` protected method
```php
...
    public $name;
    public $email;

    protected $rules = [
        'name' => 'required|min:6',
        'email' => 'required|email',
    ];

    public function updated($propertyName)
    {
        $this->validateOnly($propertyName);
    }
...
```

#### With a Normal validation
```php
...
public $name;
public $email;

public function updated($fields)
{
   $this->validateOnly($fields, [
        'name' => 'required|min:6',
        'email' => 'required|email',
     [], [
       'name' => 'Username',
       'email' => 'Email Address',
    ]);
}
...
``` 
**Context**
- Livewire version: ^2.0
- Laravel version: ^8.0

**Comment by roni-estein** [comment]
There is a var you can set I think it’s called componentAttributes, you have to search or source dive that one.

**Reply by ousid** [reply]
I can't find `component Attributes` section in the docs, Any Links I'll be appreciated
